test/testenv/turtles.go

```diff
@@ -291,92 +276,6 @@ type UpgradeRancherTurtlesInput struct {
 	SkipCleanup bool
 }
 
-// UpgradeRancherTurtles upgrades the rancher-turtles chart.
```

Comment:
PR looks good, thanks for cleanup! I only have one question: why we need to remove this func and how it is related to current cleanup?

Reply:
The setup can be simplified and we can upgrade using `DeployRancherTurtles()` since it does `helm upgrade --install`